str0m/src/stats.rs

```diff
@@ -0,0 +1,231 @@
+use itertools::Itertools;
+use std::{
+    collections::{HashMap, VecDeque},
+    time::{Duration, Instant},
+};
+
+use crate::{media::Source, Mid, Rtc};
+use rtp::Rid;
+
+pub struct Stats {
+    last_snapshot: StatsSnapshot,
+    events: VecDeque<StatEvent>,
+}
+
+type Bytes = u64;
```

Comment:
This will be confusing for readers since a lot of people are familiar with the `bytes` crate. Let's just use u64.